src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1585,6 +1617,9 @@ namespace ReportSizingManager {
                                 AutosizeDes = FinalZoneSizing(CurZoneEqNum).DesCoolMassFlow *
                                               (PsyHFnTdbW(CoilInTemp, CoilInHumRat) - PsyHFnTdbW(CoilOutTemp, CoilOutHumRat));
                                 DesVolFlow = FinalZoneSizing(CurZoneEqNum).DesCoolMassFlow / StdRhoAir;
+                                // add fan heat to coil load
+                                FanCoolLoad += DataAirSystems::calcFanDesignHeatGain(DataFanEnumType, DataFanIndex, DesVolFlow);
```

Comment:
Should this one be = instead of +=?

Reply:
FanCoolLoad = 0 at top of function so using = or += results in the same thing. Only place it matters is for air loops where a return fan could be present.